tests/test_client.py

```diff
@@ -15,10 +19,26 @@
     MicrogridApiClient,
 )
 
-from .util import patch_client_class
+from .util import ApiClientTestCaseSpec, get_test_specs, patch_client_class
 
 # pylint: disable=protected-access
 
+TESTS_DIR = Path(__file__).parent / "client_test_cases"
+
+
+@pytest.fixture
+async def client() -> AsyncIterator[MicrogridApiClient]:
```

Comment:
does it have to be async?

Reply:
Yes, because of the `async with client:` inside.